scripts/validate_response.py

```diff
@@ -11,6 +11,7 @@
 from pandas import DataFrame
 from scipy.spatial.distance import cosine, euclidean
 
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
it's interesting that `make verify` is happy with statement before import, but ok ;)